### PR TITLE
Serve DevTools from Dart SDK via DDS instead of pub

### PR DESCRIPTION
**Pull request by bkonyi**
Removes dependencies on devtools_server and devtools.

Fixes https://github.com/dart-lang/webdev/issues/1474

### CODE REVIEW

**Comment by bkonyi** [comment]
FYI @kenzieschmoll

File: dwds/test/fixtures/utilities.dart [hunk 1]
```diff
@@ -29,6 +29,7 @@ final String _sdkDir = (() {
 
 final String dartSdkPath = _sdkDir;
 final String dartPath = p.join(_sdkDir, 'bin', 'dart');
+final String devToolsPath = p.join(_sdkDir, 'bin', 'resources', 'devtools');
```
Comment (kenzieschmoll) [inline dwds/test/fixtures/utilities.dart:32]: just want to verify - will this path be accurate in g3?

Comment (bkonyi) [reply dwds/test/fixtures/utilities.dart:32]: I'm not sure. How do we even launch DevTools in google3? Where do the binaries live?

Comment (kenzieschmoll) [reply dwds/test/fixtures/utilities.dart:32]: on the dart web side, that would be a question for @annagrin. We launch devtools differently for flutter apps in google3.

Comment (annagrin) [reply dwds/test/fixtures/utilities.dart:32]: I am going to update dwds in g3 so @bkonyi can try this change out more easily. Will let you know when it is ready!

Comment (annagrin) [reply dwds/test/fixtures/utilities.dart:32]: Ben verified that the changes work with ddr (with some updates to the build and server code) so I am resolving this.

**Review by annagrin — APPROVED** [review]
LGTM! (but needs a sync and passing tests)

**Pull request merged**